docs/content/Header.mdx

```diff
@@ -54,25 +54,25 @@ All items directly under the Header component should be a `Header.Item` componen
 </Header>
 ```
 
-## Component props
+## Props
 
-### Header
+### `Header`
```

Comment:
We don't use `` for these headings in any other places. We should probably be consistent here.
```suggestion
### Header
```

Reply:
Swapped it back, but it feels pretty ugly as the other headings include object syntax